exporter/googlecloudexporter/testdata/config.yaml

```diff
@@ -24,6 +24,7 @@ exporters:
       skip_create_descriptor: true
       endpoint: test-metric-endpoint
       use_insecure: true
+      cumulative_normalization: false
```

Comment:
@dashpole this look right? It got the test to pass. I'm assuming we want to test non-default here

Reply:
Yep.  LGTM